scp/android/app/build.gradle

```diff
@@ -13,12 +13,12 @@ if (flutterRoot == null) {
 
 def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
 //if (flutterVersionCode == null) {
-    flutterVersionCode = '37'
+    flutterVersionCode = '41'
 //}
 
 def flutterVersionName = localProperties.getProperty('flutter.versionName')
 //if (flutterVersionName == null) {
-    flutterVersionName = '3.7'
+    flutterVersionName = '4.1'
```

Comment:
Okay for now, but prefer to change these versions only when doing a release at a single go and not while sending PR to master. If you have an older version on the local system and want to test a newer version, pull from master the latest before working.

Reply:
The version in the playstore was 4.1 and it was not pushed to the repo since we published the change from the local system. 
The version number was changed here to make it even with our release.